pokemonEntryTemplate.phtml

```diff
@@ -0,0 +1,17 @@
+<li class="poke-container">
+    <div class="info">
+        <div class="upper">
+            <h2 class="poke-title"><?php echo $pokemon->getName(); ?></h2>
+            <h2 class="poke-id">#<?php echo $pokemon->getId(); ?></h2>
+        </div>
+        <div class="lower">
+            <div class="type-holder">
+                <p class="type"><?php echo $pokemon->getType1(); ?></p>
+                <p class="type"><?php echo $pokemon->getType2(); ?></p>
+            </div>
+        </div>
```

Comment:
You might want to include the icons and radio buttons that appear on the wireframe, since they are part of this story.